### ISSUE TITLE
Code Table Request - Agent category

### ISSUE THREAD

**Issue by aklompma** [ArctosDB/arctos#4586]
**Goal**
Add agent category "Discoverer"

**Context**
When finding fossils or any organisms/object, the Collector (one who picks it up) does not need to be the same person as the one who found it. A new agent type "Discoverer" is proposed, unless another term sounds better (Finder?)? 

**Table**
https://arctos.database.museum/info/ctDocumentation.cfm?table=ctagent_type

**Proposed Value**
Discoverer

**Proposed Definition**
Person who discovered or found the organism, fossil, rock, mineral, artefact, object, etc. Does not need to be the same as the collector.

**Priority**
Medium

**Comment by AJLinn** [comment]
I think this activity would best be documented through a new [specimen/event](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_event_type) that could be observation or encounter, if they didn't collect it at the time. Then you could be very specific about what that person did and when and where.

This approach would provide more specific information than adding a new agent role with a potentially problematic term.

**Comment by dustymc** [comment]
Agree with @AJLinn

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecimen_event_type#observation would be the appropriate event type for "found, didn't do anything about it."

Additional record(s) could document collecting (and whatever else is necessary).

Any or all of that could all be lumped together in one record or cataloged separately and grouped with an [Entity](https://handbook.arctosdb.org/documentation/entity.html). Which is "correct" probably depends on your interpretation of how important the association between parts and events is, at the moment I'd probably recommend Entities+multiple cataloged items because I think that provides the least possibility for confusion.

That still doesn't completely address https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcollector_role, but I think it's just our definitions which need adjusted - the "finder" was the [collector](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcollector_role#collector) of the observation.

**Comment by Jegelewicz** [comment]
@AJLinn @dustymc yes - that definitely makes more sense! Plus then you could see that the fossil was discovered on 2022-04-01 but collected on 2022-04-11. I don't know if the entity creation is really necessary here? Including both events in a single record is kinda like how cultural collections have 3 events for their stuff. I also know that there are a lot of ALMNH fossil records that include "found by" and maybe some at NMMNH too? @Nicole-Ridgwell-NMMNHS These should probably all get similar treatment.

**Comment by dustymc** [comment]
>don't know if the entity creation is really necessary here?

Yea, me either, but users should be aware that it's available - maybe some big fossil that takes years to get out should for whatever reason be treated more like a wolf with multiple blood samples.

I think this may be (somehow) related to "[site found](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctlocality_attribute_type#site_found)" as well - @Nicole-Ridgwell-NMMNHS ??

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
I agree that an observation specimen event would work well. We usually use the site found locality attribute because it works better with how our data is organized (I usually don't know which particular specimen the discoverer happened to see before they called in a researcher to investigate and collect a site).

**Comment by Jegelewicz** [comment]
@aklompma did we get you an answer? If so, I'd love to see this in action so we have an example for documentation.

**Comment by aklompma** [comment]
Thanks! I am fine with adding an observation event in addition to a collecting event (assuming not all locality and stratigraphic data needs to duplicated). 
Have not been successful in adding an observation event for this one so far.
https://arctos.database.museum/guid/ALMNH:Paleo:7008

**Comment by Jegelewicz** [comment]
> assuming not all locality and stratigraphic data needs to duplicated)

It should be, but that is just a single pick from locality. Just use the one that already exists for the other event.

**Comment by Jegelewicz** [comment]
@aklompma maybe the easiest thing is for us to meet up and add it together? I'm about to eat some lunch. Are you free this afternoon?

**Comment by aklompma** [comment]
@Jegelewicz Teaching for the rest of the afternoon, starting in 30 mins. Can do it next week on Tuesday (noon CST) or Wednesday (10 am CST).

**Comment by Jegelewicz** [comment]
Sent a meeting invite!

**Comment by dustymc** [comment]
Resolved?
